ts/features/bonus/siciliaVola/screens/voucherList/VoucherListScreen.tsx

```diff
@@ -0,0 +1,204 @@
+import { View } from "native-base";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../../store/reducers/types";
+import I18n from "../../../../../i18n";
+import {
+  BottomTopAnimation,
+  LightModalContext
+} from "../../../../../components/ui/LightModal";
+import SvVoucherListFilters from "../../components/SvVoucherListFilters";
+import {
+  svPossibleVoucherStateGet,
+  svResetFilter,
+  svVoucherListGet
+} from "../../store/actions/voucherList";
+import ListItemComponent from "../../../../../components/screens/ListItemComponent";
+import ItemSeparatorComponent from "../../../../../components/ItemSeparatorComponent";
+import { svVouchersSelector } from "../../store/reducers/voucherList/vouchers";
+import { toArray } from "../../../../../store/helpers/indexer";
+import { formatDateAsLocal } from "../../../../../utils/dates";
+import {
+  FilterState,
+  svFiltersSelector
+} from "../../store/reducers/voucherList/filters";
+import { VoucherPreview } from "../../types/SvVoucherResponse";
+import {
+  svRequiredDataLoadedSelector,
+  svVouchersListUiSelector
+} from "../../store/reducers/voucherList/ui";
+import {
+  isLoading,
+  isReady,
+  isUndefined
+} from "../../../bpd/model/RemoteValue";
+import { svGenerateVoucherStart } from "../../store/actions/voucherGeneration";
+import { useIODispatch } from "../../../../../store/hooks";
+import { confirmButtonProps } from "../../../bonusVacanze/components/buttons/ButtonConfigurations";
+import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
+import EmptyListImage from "../../../../../../img/bonus/siciliaVola/emptyVoucherList.svg";
+import { InfoScreenComponent } from "../../../../../components/infoScreen/InfoScreenComponent";
+import { LoadingErrorComponent } from "../../../bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent";
+import { possibleVoucherStateSelector } from "../../store/reducers/voucherList/possibleVoucherState";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const RenderItemBase = (voucher: VoucherPreview): React.ReactElement | null => (
+  <ListItemComponent
+    title={voucher.destination}
+    subTitle={formatDateAsLocal(voucher.departureDate, true, true)}
+    onPress={() => true}
+  />
+);
+
+/**
+ * In order to optimize the rendering of the item, we use the keyId as unique identifier to avoid to redraw the component.
+ * The trx data cannot change while consulting the list and we use this information to avoid a deep comparison
+ */
+export const RenderItem = React.memo(
+  RenderItemBase,
+  (prev: VoucherPreview, curr: VoucherPreview) =>
+    prev.idVoucher !== curr.idVoucher
```

Comment:
```suggestion
    prev.idVoucher === curr.idVoucher
```
This should be the condition that identify the same props

Reply:
Thank you, fixed in [6ee7b83](https://github.com/pagopa/io-app/commit/6ee7b834c50f344f9a77ca15800de3b4e740701e)